plopfile.js

```diff
@@ -59,7 +64,7 @@ export default function (
       },
       {
         type: "add",
-        path: "src/domains/{{kebabCase name}}/queries/useSomethingQueryExample.tsx",
+        path: "src/domains/{{kebabCase name}}/queries/{{kebabCase name}}.tsx",
```

Comment:
If going to implement the change of the hooks being defined modular instead of in a big hoook this name is fine, else I would like even more a naming like "use{{pascalCase name}}.tsx